<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardConsoleClass</class>
 <widget class="QWidget" name="BoardConsoleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>975</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BoardConsole</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="connectGroupBox">
       <property name="title">
        <string>STM</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="serialComboBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="connectButton">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="filtergroupBox">
       <property name="title">
        <string>Filters</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="noFilterCheckBox">
          <property name="text">
           <string>No Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kalmanFilterCheckBox">
          <property name="text">
           <string>Kalman</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="averagingCheckBox">
          <property name="text">
           <string>Averaging</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="accelgroupBox">
       <property name="title">
        <string>Accelerometer</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QRadioButton" name="HZ100RadioButton">
          <property name="text">
           <string>100 Hz</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="HZ800RadioButton">
          <property name="text">
           <string>800 Hz</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="stabGroupBox">
       <property name="title">
        <string>Stabilization</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="stabToggleButton">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calibrButton">
          <property name="text">
           <string>Calibrate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="k1Button">
          <property name="text">
           <string>K1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="k1LineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="k2Button">
          <property name="text">
           <string>K2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="k2LineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>PWM1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="pwm1SpinBox">
          <property name="minimum">
           <number>1000</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>PWM2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="pwm2SpinBox">
          <property name="minimum">
           <number>1000</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QGroupBox" name="telemetryGroupBox">
       <property name="title">
        <string>Telemetry</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="saveToFileCheckBox">
          <property name="text">
           <string>Save to file</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="fullRadioButton">
              <property name="text">
               <string>Full</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="moveRadioButton">
              <property name="text">
               <string>Move</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="axRadioButton">
              <property name="text">
               <string>AX</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="ayRadioButton">
              <property name="text">
               <string>AY</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="azRadioButton">
              <property name="text">
               <string>AZ</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="angleCheckBox">
              <property name="text">
               <string>Angle</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="angVelCheckBox">
              <property name="text">
               <string>AngVel</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="fCheckBox">
              <property name="text">
               <string>F</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="pwm1CheckBox">
              <property name="text">
               <string>pwm1</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="pwm2CheckBox">
              <property name="text">
               <string>pwm2</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="telemetryToggleButton">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QwtPlot" name="plot1"/>
     </item>
     <item>
      <widget class="QwtPlot" name="plot2"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="boardconsole.qrc"/>
 </resources>
 <connections/>
</ui>
